FieldOpt/Settings/optimizer.h

```diff
@@ -82,6 +82,11 @@ class Optimizer
   struct Objective {
     ObjectiveType type; //!< The objective definition type (e.g. WeightedSum)
     bool use_penalty_function; //!< Whether or not to use penalty function (default: false).
+    bool use_well_cost; //!<Whether to include a well cost associated with the length
+    bool separatehorizontalandvertical;
```

Comment:
You should add some more `//!< Doxygen comments` here, e.g. is the cost pr. meter?